lib/Sema/CSDiagnostics.cpp

```diff
@@ -4796,7 +4796,9 @@ bool MissingArgumentsFailure::diagnoseClosure(const ClosureExpr *closure) {
 
   auto *locator = getLocator();
   if (locator->isForContextualType()) {
-    funcType = getContextualType(locator->getAnchor())->getAs<FunctionType>();
+    if (auto type = getContextualType(locator->getAnchor())) {
```

Comment:
This is interesting - do you have an example which caused a crash due to missing null check?

Reply:
This is something the stress tester found while I migrated to type check multi-statement closure bodies inside AST-transformed result builders. I’ll remove this check and 
```swift
  if (!getFromType()->is<TupleType>() || !getToType()->is<TupleType>()) {
     return false;
   }
```

and check whether we actually need them here.